daemons/gptp/common/ieee1588port.cpp

```diff
@@ -598,7 +598,8 @@ void IEEE1588Port::processEvent(Event e)
 
 	case STATE_CHANGE_EVENT:
 		if (!automotive_profile) {       // BMCA is not active with Automotive Profile
-			if ( clock->getPriority1() != 255 ) {
+			if ( clock->getPriority1() != 255 && qualified_announce != NULL ) {
```

Comment:
This line was added due to null pointer faults (gPTP crashes) we observed in-house. I see that EBest checks were added here on the same day, so perhaps additional unnecessary checks were added. I'll update with this commit removed.